deeppavlov/core/trainers/nn_trainer.py

```diff
@@ -73,6 +73,19 @@ class NNTrainer(FitTrainer):
         max_test_batches: maximum batches count for pipeline testing and evaluation, overrides ``log_on_k_batches``,
             ignored if negative (default is ``-1``)
         **kwargs: additional parameters whose names will be logged but otherwise ignored
+
+
+    Trainer saves the model if it sees a progress in scores. The full rules look like following:
+
+    - For the validation savepoint:
+      * 0-th validation (optional). Don't save model, establish a baseline.
+      * 1-th validation.
+         + If we have a baseline, save the model if we see an improvement, don't save otherwise.
+         + If we don't have a baseline, save the model.
+      * 2nd and later validations. Save the model if we see an improvement
+    - For the at-train-exit savepont:
+      * Save the model if it happened before 1st validation (to capture early training results), don't save otherwise.
```

Comment:
Documentation build fails here because of `Unexpected indentation`

```
    Trainer saves the model if it sees a progress in scores. The full rules look like following:

    - For the validation savepoint:
       * 0-th validation (optional). Don't save model, establish a baseline.
       * 1-th validation.
          + If we have a baseline, save the model if we see an improvement, don't save otherwise.
          + If we don't have a baseline, save the model.
       * 2nd and later validations. Save the model if we see an improvement
    - For the at-train-exit savepoint:
       * Save the model if it happened before 1st validation (to capture early training results), don't save otherwise.
```
works

Reply:
Fixed identation